eras/babbage/test-suite/cardano-ledger-babbage-test.cabal

```diff
@@ -23,6 +23,7 @@ library
     exposed-modules:
         Test.Cardano.Ledger.Babbage.Examples.Consensus
         Test.Cardano.Ledger.Babbage.Serialisation.Generators
+        Test.Cardano.Ledger.Babbage.TxInfo
```

Comment:
Addition of a new module requires a minor version bump: `1.1.1.0`

Reply:
ah, so the plan is to **_not_** do change log version for test packages, but still do cabal versioning?

Reply:
Yes versioning is more important than changelogs, without it releasing to CHaPs will not work.

If you'd like, we could do changelog for test-suites as well, but considering we are almost exclusive users of those test suites and the fact that we'll deprecate them eventually, I figured it would be an unnecessary overhead. However, we still do release them to CHaPs, cause consensus uses some parts of it, so versioning is necessary.

That is another major reason why I want to deprecate them and move the tests into the era packages themselves, to remove this overhead of separate versioning and releasing

Reply:
nope, just trying to ensure we are all agreed to a common plan. I appreciate all your help with this!